### PR TITLE
Nathan4

### PR DESCRIPTION
**Pull request by NathanP9000**
image
image

### CODE REVIEW

File: src/components/Home/index.tsx [hunk 1]
```diff
@@ -15,8 +15,10 @@ type HomeProps = {
 }
 
 export default function Home({ userInfo, isStudent, onAppointmentRequestResolve }: HomeProps) {
-  const router = useRouter()
-
+  const router = useRouter()  
+  const goAvailability = () => {
+    router.push("/availability")
+  }
```
Comment (danielkangel) [inline src/components/Home/index.tsx:21]: What's this for now

Comment (danielkangel) [inline src/components/Home/index.tsx:21]: What's this for

File: src/pages/availability/index.tsx [hunk 2]
```diff
@@ -98,6 +98,12 @@ export default function Availability() {
                         return d
                     })}><CaretIcon className="-rotate-90" /></button>
                 </div>
+                <span className="text-2xl">{monthNames[date.getMonth()]} {date.getFullYear()}</span>
+                <button 
+                className="w-30 text-white text-sm bg-[#5B7B7A] hover:bg-[#3B5B5A] rounded-lg "
```
Comment (danielkangel) [inline src/pages/availability/index.tsx:103]: Can we make this button less wide? Alsoo, maybe change the location? I think it looks a little odd where it is rn

File: src/components/Calendar/index.tsx [hunk 3]
```diff
@@ -57,7 +57,12 @@ export default function Calendar({ size, selectedDate, setSelectedDate, availabi
...
+                return <button key={Math.random()} 
+                    className={`m-auto w-12 text-xl font-xl rounded-full 
+                    ${availability.includes(new Date(date.getFullYear(), date.getMonth(), d).toDateString()) && new Date(selectedDate.getFullYear(), selectedDate.getMonth(), selectedDate.getDate()).toDateString() === new Date(date.getFullYear(), date.getMonth(), d).toDateString() ? "bg-brown-50" : ""} 
+                    ${new Date(date.getFullYear(), date.getMonth(), d) >= new Date(new Date().getFullYear(), new Date().getMonth(), new Date().getDate()) && availability.includes(new Date(date.getFullYear(), date.getMonth(), d).toDateString()) ? "hover:bg-brown-50" : "text-gray-400"} transition aspect-square ${d === 0 ? "opacity-0" : ""}`} 
+                    disabled={d === 0 || !availability.includes(new Date(date.getFullYear(), date.getMonth(), d).toDateString()) || new Date(date.getFullYear(), date.getMonth(), d) < new Date(date.getFullYear(), date.getMonth(), new Date().getDate())} 
+                    onClick={() => setSelectedDate(new Date(date.getFullYear(), date.getMonth(), d))}>{d}</button>
```
Comment (danielkangel) [inline src/components/Calendar/index.tsx:65]: Can you lint this to make it more legible? 🙏

**Comment by NathanP9000** [comment]
image

File: src/pages/availability/index.tsx [hunk 4]
```diff
@@ -82,10 +81,14 @@ export default function Availability() {
     })
     return (
         <div className="flex flex-col w-full p-8">
-            <h1 className="text-5xl font-bold">Availability</h1>
-            <div className="flex flex-row w-full gap-6 items-center p-24 py-8">
-                <span className="text-2xl">{monthNames[date.getMonth()]} {date.getFullYear()}</span>
-                <div className="flex flex-row ml-auto items-center">
+            <h1 className="flex text-5xl font-bold justify-end w-3/5">Availability</h1>
```
Comment (danielkangel) [inline src/pages/availability/index.tsx:84]: Why is this h1 and flex container

**Review by danielkangel — APPROVED** [review]

**Pull request merged**